spec/javascripts/support/jasmine_helper.rb

```diff
@@ -2,6 +2,7 @@
 
 class HeadlessChromeJasmineConfigurer < JasmineSeleniumRunner::ConfigureJasmine
   def selenium_options
-    { options: GovukTest.headless_chrome_selenium_options }
+    args = %w[no-sandbox disable-dev-shm-usage disable-gpu]
+    Selenium::WebDriver::Chrome::Options.new(args: args)
```

Comment:
I think you might need the `headless!` bit and return the same hash structure too

```
    args = %w[no-sandbox disable-dev-shm-usage disable-gpu]
    options = Selenium::WebDriver::Chrome::Options.new(args: args)
    options.headless!
    return { options: options }
```